doc/development/developing-sql-dialect/integration_testing_with_containers.md

```diff
@@ -125,12 +55,23 @@ Note that to check whether the integration-tests were successful, you have to ru
 You can start the integration tests as follows:
 
 ```bash
-mvn clean package && mvn verify -Pit -Dintegrationtest.configfile=/path/to/your/integration-test-config.yaml
+mvn clean package && mvn verify
 ```
 
-This will run all integration tests, i.e. all JUnit tests with the suffix `IT` in the filename.
+This will run all included into the `maven-failsafe-plugin` integration tests.
+
+Another way to run integration tests:
+
+* Create a package of Virtual Schemas using `mvn package` command and run integration tests inside your IDE in the same way as unit tests.
+
+## Executing Disabled Integration Tests
+
+Some integration tests are not running automatically, but it is possible to execute them locally. For example, `OracleSqlDialectIT`.
 
-The YAML configuration file stores the information for your test environment like JDBC connection strings, paths and credentials.
+In order to start `OracleSqlDialectIT`:
+* Download Oracle JDBC driver `ojdbc8.jar` and oracle instant client `instantclient-basic-linux.x64-12.1.0.2.0.zip` and temporary put them into `src/test/resources/integration/driver/oracle` directory.
+* Run the tests from an IDE or temporary add `OracleSqlDialectIT.java` into the `maven-failsafe-plugin` includes section and execute  `mvn verify` command.
+* Remove the driver and the instant client after the test. Do not upload them to the GitHub repository.
```

Comment:
Better: add everything into `.gitignore` so that people can't accidentally check those drivers in!

Reply:
Good idea!

Reply:
done